### ISSUE TITLE
YOLO Training with Combined Dataset

### ISSUE THREAD

**Issue by margoglvz** [CareTech-at-UCI/vitamin-tracking#9]
## **Getting Started**
- Create a new branch with `git checkout -b 'partner1-FN-partner2-FN'` (FN = First Name)
- Make a new folder called  `'partner1-FN_partner2-FN'`
- Using the combined dataset provided, we will ask you to train and optimize multiple models with it to report your best resulting model/s
  - Find combined dataset here: [Roboflow Combined Dataset](https://universe.roboflow.com/caretech-qnw0y/caretech-combined-dataset-kcl5q)
  - Use the provided code in the Secrets document to download the dataset. You will find it under the 'COMBINED DATASET' tab
- You can also look at the [YOLO Workshop Colab](https://colab.research.google.com/drive/16uWMIcOXW62GMm14YXN-07IincBE6lRb?usp=sharing) for help!

## **Combined Dataset**
- The Combined Dataset is made up of the UECFOOD256 and Yusuf Dataset 
  - The UECFOOD256 Dataset was converted to YOLO format so if you would like to make your own combined dataset with different preprocessing, please reach out to Margaret and Leo for more information
- Preprocessing included converting to YOLO format and removing duplicates. Normalization and resizing are **not** needed as they are taken care of with Ultralytics.

## Train/Val/Test Split
- The Combined Dataset already has its respective train/val/test with images and its respective labels
  - In case you need a refresher! Each image (e.g. jpg) has a respective text file (e.g. .txt) that contains the YOLO-formatted bounding box information
  - rice1.jpg has bbox information stored in rice1.txt

## Submission
Look at [CareTech Resources Page](https://docs.google.com/document/d/1xWkiNY6oqvcNrNZ_sTFa7KFGUolHKp4kYOogBnB2Wso/edit?usp=sharing) for full instructions on submitting a GitHub Pull Request!

Please train multiple YOLO models with the Combined Dataset and include your **top performing** models and its **hyperparameters** in the description. You may choose how many models you would like to display. Please make sure to include but not limited to these types of information:
- Number of epochs
- Learning rate
- Batch size
- mAP on validation set
- YOLO filters (if any were used)

## **DISCLAIMER**
**It is important you are confident in your models before you enter the final testing phase**. Your next task will be re-evaluating preprocessing and making sure you wanted to try out everything before you test them. So, really plan on which hyperparameters you would like to test!
